pyproject.toml

```diff
@@ -43,6 +43,10 @@ clickhouse = [
 duckdb = [
   "ibis-framework[duckdb]>=11.0.0",
 ]
+delta = [
+  "deltalake>=1.2.1",
+  "obstore>=0.8.2",
```

Comment:
obstore shouldn't be a dependency, deltalake works without it

Reply:
not even for the deletion of full tables/features?